source/index.html

```diff
@@ -1,12 +1,89 @@
+          alt="Логотип Кэт энерджи">
+      </a>
+      <nav class="nav">
+        <button class="nav__toggle">Открыть меню</button>
+        <div class="nav__wrapper">
+          <ul class="nav__menu site-menu">
+            <li class="site-menu__item site-menu__item--active">
+              <a>Главная</a>
+            </li>
+            <li class="site-menu__item">
+              <a href="#">Каталог продукции</a>
+            </li>
+            <li class="site-menu__item">
+              <a href="form.html">Подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <h1 class="visually-hidden">Кэт энерджи</h1>
+      <section class="intro">
+        <header class="intro__header">
+          <img class="intro__bg-image" src="https://via.placeholder.com/320x305" width="320" height="305"
+            alt="Изображение кота">
+          <div class="intro__wrapper">
+            <h2 class="visually-hidden">Рекламный блок Кэт энерджи</h2>
+            <b class="intro__description">Функциональное питание для котов</b>
+            <p class="intro__slogan">Занялся собой? Займись котом!</p>
+            <img class="intro__promo-image" src="https://via.placeholder.com/280x270" width="280" height="270"
+              alt="Изображение банки кошачьего питания Кэт энерджи">
+            <a href="form.html" class="button intro__button">Подобрать программу</a>
+          </div>
+        </header>
+      </section>
+      <section class="programs">
+        <div class="programs__wrapper">
+          <h2 class="visually-hidden">Наши программы</h2>
+          <ul class="programs__list">
+            <li class="programs__item programs__item--slim">
+              <h3 class="programs__title">Похудение</h3>
+              <p class="programs__description">Ваш кот весит больше собаки и почти утратил способность лазить по
+                деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+              <a href="#" class="programs__link">Каталог Slim</a>
+            </li>
+            <li class="programs__item programs__item--pro">
+              <h3 class="programs__title">Набор массы</h3>
+              <p class="programs__description">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy
+                Pro поможет вашему коту нарастить необходимые мышцы!</p>
+              <a href="#" class="programs__link">Каталог Pro</a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="advantages">
+        <div class="advantages__wrapper">
+          <h2 class="advantages__title">Как это работает?</h2>
+          <ul class="advantages__list">
+            <li class="advantages__item advantages__item--healthy">
+              <p class="advantages__text">Функциональное питание содержит только полезные питательные вещества.</p>
```

Comment:
```suggestion
              <p class="advantages__item-text">Функциональное питание содержит только полезные питательные вещества.</p>
```

Обрати внимание на похожие проблемы, и поправь их.